aea/context/base.py

```diff
@@ -126,19 +121,9 @@ def decision_maker_message_queue(self) -> Queue:
         return self._decision_maker_message_queue
 
     @property
-    def ownership_state(self) -> OwnershipState:
-        """Get the ownership state of the agent."""
-        return self._ownership_state
-
-    @property
-    def preferences(self) -> Preferences:
-        """Get the preferences of the agent."""
-        return self._preferences
-
-    @property
-    def goal_pursuit_readiness(self) -> GoalPursuitReadiness:
-        """Get the goal pursuit readiness."""
-        return self._goal_pursuit_readiness
+    def decision_maker_handler_context(self) -> SimpleNamespace:
+        """Get the decision maker handler context."""
+        return self._decision_maker_handler_context
```

Comment:
Very verbose. @marcofavorito any better naming suggestion?

Reply:
`dm_context`? We could drop `handler` because conceptually you can identify the handler with the decision-maker, since is the class that defines the DM behaviour the most.

Re. `dm`, maybe it is a reasonable abbreviation? If you ever care about the decision making... well you know what `dm` refers to.